packages/components/addon/components/hds/dropdown/list-item/checkbox.hbs

```diff
@@ -10,10 +10,16 @@
       <FlightIcon @name={{@icon}} @isInlineBlock={{false}} />
     </div>
   {{/if}}
-  <label class="hds-dropdown-list-item__label hds-typography-body-200" for={{this.id}}>
+  <Hds::Text::Body @tag="label" @size="200" class="hds-dropdown-list-item__label" for={{this.id}}>
     {{yield}}
-  </label>
+  </Hds::Text::Body>
```

Comment:
I liked the pattern Melanie shared:

```
{{#element @tag as |El|}}
  <El ...attributes>
    {{! ... }}
  </El>
{{/element}}
```

At least for more semantic elements such as `label`, `li`, etc.

Reply:
It's already the pattern we're using in the `Text` component 9slightly different syntax, but it's exactly the same):
https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/text/index.hbs#L6-L8

Reply:
I still prefer the above pattern in interests of readability but I'm ok with shipping.

Reply:
@KristinLBradley I tried that syntax but this is the error I see:
<img width="671" alt="screenshot_3091" src="https://github.com/hashicorp/design-system/assets/686239/f9f18e02-3aed-4194-95e2-1113fe2957a0">

@MelSumner any idea why?

Reply:
Hmm